plugins/outputs/sumologic/README.md

```diff
@@ -0,0 +1,66 @@
+# SumoLogic Output Plugin
+
+This plugin sends metrics to [Sumologic HTTP Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source/Upload-Metrics-to-an-HTTP-Source)
+in HTTP messages, encoded using one of the output data formats.
+Currently metrics can be sent using one of the following data formats, supported
+by Sumologic HTTP Source:
+
+  * `graphite` - for Content-Type of `application/vnd.sumologic.graphite`
+  * `carbon2` - for Content-Type of `application/vnd.sumologic.carbon2`
+  * `prometheus` - for Content-Type of `application/vnd.sumologic.prometheus`
+
+### Configuration:
+
+```toml
+# A plugin that can send metrics to Sumo Logic HTTP metric collector.
+[[outputs.sumologic]]
+  ## Unique URL generated for your HTTP Metrics Source.
+  ## This is the address to send metrics to.
+  # url = "https://events.sumologic.net/receiver/v1/http/<UniqueHTTPCollectorCode>"
+
+  ## Data format to be used for sending metrics.
+  ## This will set the "Content-Type" header accordingly.
+  ## Currently supported formats: 
+  ## * graphite - for Content-Type of application/vnd.sumologic.graphite
+  ## * carbon2 - for Content-Type of application/vnd.sumologic.carbon2
+  ## * prometheus - for Content-Type of application/vnd.sumologic.prometheus
+  ##
+  ## More information can be found at:
+  ## https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source/Upload-Metrics-to-an-HTTP-Source#content-type-headers-for-metrics
+  ##
+  ## NOTE:
+  ## When unset, telegraf will by default use the influx serializer which is currently unsupported
+  ## in HTTP Source.
+  data_format = "carbon2"
+
+  ## Timeout used for HTTP request
+  # timeout = "5s"
+  
+  ## HTTP method, one of: "POST" or "PUT". "POST" is used by default if unset.
+  # method = "POST"
+
+  ## HTTP Content-Encoding for writing request body. Can be set to "gzip" to compress the body.
+  ## This will set the "Content-Type" header accordingly.
+  # content_encoding = "gzip"
```

Comment:
Whats the default value for the `content_encoding`?

Reply:
It's an empty string (we don't set it by default) which means we don't compress by default. I guess we should change that to `gzip` by default and hide the configuration option as per Frank's comment: 

https://github.com/SumoLogic/telegraf/pull/1#discussion_r473249166